k8s/migration/pkg/utils/credutils.go

```diff
@@ -293,14 +293,28 @@ func GetOpenstackInfo(ctx context.Context, k3sclient client.Client, openstackcre
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to extract all networks")
 	}
-
+	allPagesv, err := schedulerstats.List(openstackClients.BlockStorageClient, nil).AllPages()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to extract all storage backend pools")
+	}
+	pools, err := schedulerstats.ExtractStoragePools(allPagesv)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to extract all storage backend pools")
+	}
+	volBackednPools := make([]string, 0, len(pools))
+	for _, pool := range pools {
+		if pool.Name != "" {
```

Comment:
I don't think there could be any pool without name. Were there cases where we saw pools with empty names?

If not, this can be simplified like other lists we have got above.